log.go

```diff
@@ -95,3 +95,7 @@ func (l *Logger) Warn() *Event {
 func (l *Logger) Error() *Event {
 	return l.newEvent(ErrorLevel)
 }
+
+func (l *Logger) Fatal() *Event {
+	return l.newEvent(ErrorLevel)
```

Comment:
I think this is a fatal level not an error level @salman0ansari

Reply:
oops sorry